src/app/legacy/components/RadioSchedule/ProgramCard/index.jsx

```diff
@@ -131,7 +131,7 @@ ProgramCard.propTypes = {
   program: shape({
     state: string.isRequired,
     duration: string.isRequired,
-    startTime: number.isRequired,
+    startTime: oneOfType([number, string]).isRequired,
```

Comment:
Since we have made changes to legacy should there be a refactor ticket for this now to bring it to new component standards and create typescript types for an upcoming sprint?